static/app/views/insights/browser/resources/queries/useIndexedResourceQuery.ts

```diff
@@ -17,10 +21,29 @@ export const useIndexedResourcesQuery = ({
   referrer,
   enabled = true,
 }: Options) => {
-  return useSpansIndexed(
+  const useEap = useInsightsEap();
+
+  const eapResult = useEAPSpans(
+    {
+      fields: [
+        SpanFields.PROJECT,
+        SpanFields.SPAN_GROUP,
+        SpanFields.RAW_DOMAIN,
+        SpanFields.SPAN_DESCRIPTION,
+        SpanFields.MEASUREMENT_HTTP_RESPONSE_CONTENT_LENGTH,
```

Comment:
are these fields for an eap specific change?

Reply:
These are the same fields as before,  i just added them into the new `SpanFields` type, which will be the final type once we fully migrate to eap. 

Wherever we use `useEapSpans`, i think it makes sense to use properties from `SpanFields`